app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,16 @@
+class FriendshipsController < ApplicationController
+  def create
+    @friend = User.find_by(handle: params[:follower])
+    params[:friend_id] = @friend.id
+    friendship = Friendship.create(friendship_params)
```

Comment:
Should this be a mutual friendship?

Reply:
great point. Hopefully we can get some clarity as we go along, either from Ian or from a future user story.